doc/Interns/furahadamien/MyFirstCommit.md

```diff
@@ -0,0 +1 @@
+05/06/2020
```

Comment:
Forgot to mention in the instructions.  Please add Time to this time... Date and Time is what we are looking for.

Reply:
will work on that